### ISSUE TITLE
d3d12game_uwp_cppwinrt_dr doesn't run on Xbox One S through UWP

### ISSUE THREAD

**Issue by BMarques** [walbourn/directx-vs-templates#62]
Hello,
I tried to run the [d3d12game_uwp_cppwinrt_dr](https://github.com/walbourn/directx-vs-templates/tree/main/d3d12game_uwp_cppwinrt_dr) project on my Xbox One S on Game Mode and it's not working because it says it doesn't support Shader Model 6.0. When I debugged, it says it only supports Shader Model 5.1.
Is this an hardware limitation (therefore needing to change the project to make it work) or is there something I can do about it?

**Comment by walbourn** [comment]
Hmm... For DX12, Shader Model up through 6.4 should work on Xbox One S in Game Mode. Can you confirm you are not using the WARP device?

**Comment by BMarques** [comment]
Yes, I'm not using the WARP mode (It was before when I was on App mode by mistake).
It gives me the following message for the GetAdapter.
**"Direct3D Adapter (0): VID:0000, PID:0000 - SraKmd"**

It also supports FEATURE_LEVEL 12_0 from what I could debug.

**Comment by walbourn** [comment]
What version of the OS recovery do you have on that system?

**Comment by BMarques** [comment]
Not sure what is the os recovery, but in the dev mode it says OS Version 10.0.22621.1836. I'm using UWP for Xbox, not a closed GDK

**Comment by BMarques** [comment]
Just to let you know, once I "undid" everything that was documented on https://github.com/microsoft/DirectXTK12/wiki/Shader-Model-6, both on the game project and DirectXTK12 project, the SpriteBatch tutorial of DirectXTK12 worked properly on the console

**Comment by walbourn** [comment]
OK, so I double-checked and there are some known issues with the reporting on UWP on Xbox One. It's supposed to be reporting support for Feature Level 12.0 and Shader Model 6.4, but instead it's reporting only Shader Model 5.1.

I'll ping the graphics team on this. In the meantime, you can use 5.1.
